service/test/controllers/systemsettings/SystemSettingsControllerTest.java

```diff
@@ -0,0 +1,146 @@
+package controllers.systemsettings;
+
+import static controllers.TestUtil.mapToJson;
+import static org.junit.Assert.assertEquals;
+import static org.powermock.api.mockito.PowerMockito.when;
+import static play.test.Helpers.route;
+
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+import akka.actor.Props;
+import com.fasterxml.jackson.databind.JsonNode;
+import controllers.BaseController;
+import controllers.DummyActor;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.HeaderParam;
+import play.libs.Json;
+import play.mvc.Http.RequestBuilder;
+import play.mvc.Result;
+import play.test.FakeApplication;
+import play.test.Helpers;
+import util.RequestInterceptor;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(RequestInterceptor.class)
+@PowerMockIgnore("javax.management.*")
+public class SystemSettingsControllerTest {
```

Comment:
Create a BaseControllerTest which has startApp and beforeTest methods to avoid code duplication across controller tests.